sort/countingSort.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import "fmt"
+
+//use this for strings
+func countingSortString(data []rune, size int) string {
+	length := 256
+	count := make([]int, length)
+	output := make([]rune, size)
+	for i := 0; i < size; i++ {
+		count[data[i]]++
+	}
+
+	for i := 1; i < length; i++ {
+		count[i] += count[i-1]
+	}
+
+	for i := 0; i < size; i++ {
+		output[count[data[i]]-1] = data[i]
+		count[data[i]]--
+	}
+
+	return string(output)
+}
+
```

Comment:
Please rename this function to "CountingSort" so that this function is public.
Also add proper comment. Please a take at other sorting algorithms to examples.